pkg/envinfo/envinfo.go

```diff
@@ -122,20 +132,20 @@ type EnvInfoLink struct {
 
 // getEnvInfoFile first checks for the ENVINFO variable
 // then we check for directory and eventually the file (which we return as a string)
-func getEnvInfoFile(envDir string) (string, error) {
+func getEnvInfoFile(envDir string) (string, string, error) {
 	if env, ok := os.LookupEnv(envInfoEnvName); ok {
-		return env, nil
+		return env, filepath.Join(env, "..", "..", "..", "devfile.yaml"), nil
```

Comment:
why not directly set the devfilePath in newEnvSpecificInfo() using envDir there?

Reply:
Because then I would need to duplicate the logic in that function to compute the starting point.

Reply:
I didn't get what you mean by starting point. In most cases, `envDir` is the context directory. We can just join the default devfilePath i.e `./devfile.yaml` to it. Or am I missing something?

Reply:
I may be missing something but the starting point from which the devfile path is computed depends on several things. Look at the code for this function and you'll see what I mean.

Reply:
That said, I've seen different ways of dealing with this in different parts of the code so… 🤷

Reply:
I checked the function again. You are right, my suggestion won't work in case of the env specified envDir.